garage/tf/algos/batch_polopt.py

```diff
@@ -50,9 +56,14 @@ def __init__(self,
         self.center_adv = center_adv
         self.positive_adv = positive_adv
         self.fixed_horizon = fixed_horizon
+
+        self.eprewmean = deque(maxlen=100)
```

Comment:
please use a better variable name for this -- we don't pay for code hosting by the character :P. `self.episode_reward_mean`